test/oracles/BalancerWeightedPoolPriceOracle/Parameters.sol

```diff
@@ -1,9 +1,8 @@
 // SPDX-License-Identifier: GPL-3.0
 pragma solidity >=0.8.0 <0.9.0;
 
-import "forge-std/Test.sol";
+import {Test} from "forge-std/Test.sol";
```

Comment:
Are we sure we want to be explicit on importing only that required here for the `forge-std/Test.sol` (noting your standards that you've arrive at which are solid).

There might be a slight foot-gun here in that this then wouldn't import `console`, and perhaps something that's easily forgotten during the course of development?

Reply:
This was the only one I was considering making an exception, on the other hand it's fairly easy to change it to `import {Test, console}`.
My very weak preference is making this explicit as well, but if you feel it's annoying during development I don't mind changing it.

Reply:
I think I can get used to it (importing explicitly) :sweat_smile: